components/display/Inkplate.rst

```diff
@@ -0,0 +1,247 @@
+Inkplate 6
+==========
+
+.. seo::
+    :description: Instructions for setting up Inkplate E-Paper displays in ESPHome.
+    :image: Inkplate.jpg
+
+All-in-one e-paper display  ``Inkplate 6``
+Inkplate 6 is a powerful, Wi-Fi enabled ESP32 based six-inch e-paper display – recycled from a Kindle e-reader. Its main feature is simplicity.
+Learn more at `our website <https://inkplate.io/>`__
```

Comment:
Please write `inkplate's` website instead. I don't want to give the impression that ESPHome is somehow directly related to inkplate (users viewing the docs won't look who made the docs PR)

Reply:
Done that :D